Assets/Python/Contrib/PythonToolTip.py

```
@@ -6,228 +6,286 @@
+        iWidth = CyIF.determineWidth(uFont + aLine)
+        aWidthList.append((iWidth, aLine))
+        if iWidth > dx:
+            dx = iWidth
+
+    # Clean up temporary list immediately
+    del aList
+
+    if dx > dxMax:
+        dx = dxMax
+    elif not dx:
+        print
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)